engine/src/test/java/org/hibernate/validator/test/internal/constraintvalidators/hv/URLValidatorTest.java

```diff
@@ -363,22 +364,16 @@ private void assertValidUrls(ConstraintValidator<URL, CharSequence> validator) {
 			assertFalse( validator.isValid( "http:///a", null ) );
 			assertFalse( validator.isValid( "http:// shouldfail.com", null ) );
 			assertFalse( validator.isValid( "http://foo.bar/foo(bar)baz quux", null ) );
-//			assertFalse( validator.isValid( "http://-error-.invalid/", null ) );
-//			assertFalse( validator.isValid( "http://a.b--c.de/", null ) );
-//			assertFalse( validator.isValid( "http://-a.b.co", null ) );
-//			assertFalse( validator.isValid( "http://a.b-.co", null ) );
-//			assertFalse( validator.isValid( "http://0.0.0.0", null ) );
-//			assertFalse( validator.isValid( "http://10.1.1.0", null ) );
-//			assertFalse( validator.isValid( "http://10.1.1.255", null ) );
-//			assertFalse( validator.isValid( "http://224.1.1.1", null ) );
-//			assertFalse( validator.isValid( "http://1.1.1.1.1", null ) );
+			assertFalse( validator.isValid( "http://-error-.invalid/", null ) );
+			assertFalse( validator.isValid( "http://a.b--c.de/", null ) );
+			assertFalse( validator.isValid( "http://-a.b.co", null ) );
+			assertFalse( validator.isValid( "http://a.b-.co", null ) );
```

Comment:
yes, it's this change:
`DOMAIN_LABEL = "(" + DOMAIN_CHARS_WITHOUT_DASH + "-?)*" + DOMAIN_CHARS_WITHOUT_DASH + "+"`